src/core/Clock.js

```diff
@@ -12,9 +12,15 @@ class Clock {
 
 	}
 
+	getPerformanceNow() {
```

Comment:
I don't think we should expose this method. 

If you want to avoid duplicate code, consider to make a function `now()` in the `Clock.js` module. A method is not necessary.